aea/aea.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, name: str,
         self._directory = directory if directory else str(Path(".").absolute())
 
         self.mailbox = mailbox
-        self._context = AgentContext(self.name, self.outbox)
+        self._context = AgentContext(self.name, self.crypto.public_key, self.outbox)
```

Comment:
We continue being very careful what we assign to the agent context. We don't want skills to get too many capabilities.